clang/test/SemaCXX/attr-lifetimebound.cpp

```diff
@@ -31,6 +32,11 @@ namespace usage_ok {
     return *(int*)param;
   }
 
+  template <class T, class R = void> R dependent_void(const T& t [[clang::lifetimebound]]);
+  void dependent_void_instantiation() {
+    dependent_void<int>(1);
```

Comment:
nit: add the following for completeness.
```
int x = dependent_void<int, int>(1); // warning.
dependent_void<int, int>(1); // Ok.
```

Reply:
Done.